examples/cd_distillation_cifar10.ipynb

```diff
@@ -30,17 +30,9 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    y_pred = predict_batch(clf, X_test, batch_size=256, return_class=True)
</pre><p> </p><p>Returns: </p><pre class="ql-syntax" spellcheck="false">TypeError: predict_batch() got an unexpected keyword argument 'return_class'
</pre><p> </p><p>See first bullet point in #323 (predict_batch args were changed when made tensorflow/pytorch specific).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Somehow missed that, this is why we need notebook tests!